### ISSUE TITLE
Old invalid abilities present even if caldera restarted

### ISSUE THREAD

**Issue by ramus82** [mitre/caldera#780]
Hello,
I attempted to create a new ability in stockpile plugin. After first version I ran the caldera instance and there was a message in Caldera WebUI (when clicking on Abilities and selected proper category): "No ability file found!". I made a type in the id of the ability. Then I made another version (with the same mistake - another typo). I succeeded on third attempt. 

Currently I can see the ability (is listed) even with correct content of the file. However the two incorrect attempts are still visible. I can see in total three abilities with the very same name (two with message "No ability file found!" and one with the correct content.

How can I get rid of the abilities?

**Comment by privateducky** [comment]
We've noticed this being a pain when creating abilities via the webUI. In the bleeding-edge code, we've since disabled this and require new abilities to be created by adding YML files into the stockpile/data/abilities directory.

To solve the problem you have, you need to remove the bad ability files (from the data/abilities/ directory) and then restart CALDERA.

**Comment by ramus82** [comment]
Thanks for quick reply, privateducky
I created the ability by uploading YML file (unfortunatelly the ID string inside of the YML file was different than filename). 

I removed the YML file I've created and restarted caldera. I can see still three abilities (the 2 incorrectly created and the 1 for the file I just removed).

I know that I made a mistake with retyping of 8 -> B and 6 -> b. When I insert the part I'm sure was the same for all three variants of the ability YML file, I can see them in the "Abilities" page.

Any other hints how to reset to default?
